src/api/search/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
-    "@senecacdot/satellite": "^1.x",
+    "@senecacdot/satellite": "^1.24.0",
```

Comment:
We don't want to do this.  I want to have pnpm use the latest version `1.x` of Satellite, so it always auto-updates.  I think you did this locally so it would pick up that change, but it should do it automatically.

Reply:
Hi Prof, I just want to make sure if this means my PR should only have the test file changed, and that the `search/package.json` and `pnpm-lock` should not be touched? 
I couldn't pick up the new versions locally when I ran `pnpm install` in root, or in the search directory. Everything in these two files that's changed now is because I ran `pnpm update @senecacdot/satellite` in the search directory. 
- Do I keep what has been changed in the filies, but follow what's been written in the comments above, manually change it to `^1.x` in the `search/package.json` and then run `pnpm install`? This way, the line above and line 273 in `pnpm-lock` will be changed back. However, the other changes in `pnpm-lock` will stay.
- Or do I revert all the changes in these two files, and submit this PR with only the test file? The tests will fail for me locally this way because its still on Satellite version `1.17` for me locally. I'm not sure if it'll pass CI

Reply:
Basically, don't change satellite's version in any `package.json` file.  It should grab the right one.  If it doesn't, delete your `node_modules` so it does.